src/equations/compound-interest.js

```diff
@@ -1,12 +1,37 @@
-export const compoundMaturity = (
-  principal,
-  interestRate,
-  periods
-) => {
-  // compoundsPerYear
+const pow = Math.pow
+const exp = Math.exp
 
-  return principal * Math.exp(interestRate*periods);
+export default class Config {
+  static title = "Compound Interest"
+  static description = "Standard PeRT Equation."
 
+  static params = {
+    principal: {
+      label: "Principal",
+      default: 1000,
+      type: "currency",
+      min: 0,
+      max: Number.POSITIVE_INFINITY,      
+    },
+    periods: {
+      label: "Periods",
+      default: 10,
+      type: "currency",
+    },
+    rate: {
+      label: "Interest Rate (Annual)",
+      default: 0.029,
+      type: "currency",
```

Comment:
same as above; this isn't a currency. this is a `rate`. we need to create these new field types. you'll also need to set a min/max here, too

Reply:
i was trying to hit that if conditional, will see about making that rate.  Rate i think is a non-complicated topic since it has both a % and a time period.

Reply:
double checked. This doesn't render anything unless the type is currency.  I'll pump the breaks on adding this until I add a separate 'number' type.